code/game/objects/items/storage/toolbox.dm

```diff
@@ -97,39 +97,6 @@
 /obj/item/storage/toolbox/mechanical/old/heirloom/PopulateContents()
 	return
 
-/obj/item/storage/toolbox/mechanical/old/clean // the assistant traitor toolbox, damage scales with TC inside
-	name = "toolbox"
-	desc = "An old, blue toolbox, it looks robust."
-	icon_state = "oldtoolboxclean"
```

Comment:
keeping the icon state?

Reply:
yeah, should remove this

Reply:
im pretty sure its used in a few places or maps

Reply:
It's not, it's safe to remove.